docsrc/admin/overview.md

```diff
@@ -1,22 +1,21 @@
 (overview)=
-
 # Overview
 
 This section gives an overview of the components of the Data Library system.
 
 (infrastructure)=
-
 ## Infrastructure
 
-The Data Library ansible playbook currently targets CentOS7 and CentOS Stream 9.
-IRI does not test or support the software on other
-platforms. CentOS7 is at its End of Life, and is no longer recommended to
-install.
+The Data Library ansible playbook currently targets [CentOS Stream 9](https://www.centos.org/stream9/). IRI does not test or support 
+the software on other platforms.
+
+_CentOS7 is beyond its end of life and is no longer recommended to install, but we continue to support the 
+installations where it is already installed. Any new installations must be made with CentOS Stream 9_
 
-The Data Library services runs under Docker, using docker compose. Most services
-log to stdout, which the Docker daemon
+The Data Library services runs under Docker, using docker compose. Most services log to stdout, which the Docker daemon
 forwards to journald.
```

Comment:
Note the mention of journald. Need to make it true, or failing that, change this doc.